ui-tests/seo-main.spec.ts

```diff
@@ -0,0 +1,9 @@
+import test, { expect } from "@playwright/test";
+
+test("No index header not present", async ({ page }) => {
+  const response = await page.goto("/", {
+    waitUntil: "networkidle",
+  });
+  expect(response.ok()).toBeTruthy();
+  expect(response.headerValue("X-Robots-Tag")).not.toEqual("noindex");
```

Comment:
should this instead check that there is no `X-Robots-Tag` set?
it may make the intent more clear

Reply:
Done - as per https://playwright.dev/docs/api/class-request#request-header-value I am now checking if the header value is falsey (null)